backend/experiment/migrations/0054_migrate_block_content.py

```diff
@@ -5,6 +5,17 @@
 from pathlib import Path
 
 
+def get_rules(block):
```

Comment:
Not too sure if this is necessary. I'm still thinking that faulty database values are at the root of this.

Reply:
https://uva-aml.sentry.io/share/issue/3d054044a62b442a89fe5e7d8f9773f7/ The error is literally `'Block' object has no attribute 'get_rules'` though